boards/arm/mps2/doc/mps2_armv7m.rst

```diff
@@ -1,14 +1,15 @@
-.. _mps2_an385_board:
+.. _mps2_armv7m_board:
 
-ARM V2M MPS2 AN385
-##################
+ARM V2M MPS2 Armv7-m (AN385/AN386/AN500)
+########################################
 
 Overview
 ********
 
-The ``mps2/an385`` board target is used by Zephyr applications that run on
-the V2M MPS2 board. It provides support for the ARM Cortex-M3 (AN385) CPU and
-the following devices:
+The mps2 board target is used by Zephyr applications that run on
```

Comment:
This is not correct, see https://docs.zephyrproject.org/latest/glossary.html#term-board-target for terms `mps2` is a board, `mps2/an385` is a board target, it would need updating throughout